docs/modules/comments.md

```diff
@@ -0,0 +1,464 @@
+---
+id: comments
+title: Comments
+---
+
+# Comments Module
+
+The reSolve comments module ([@resolve-js/module-comments](https://www.npmjs.com/package/@resolve-js/module-comments)) adds support for hierarchical user comments in a reSolve application. In addition to aggregates and read models that describe the comments logic on the server side, the module exports renderless client components based on React + Redux and the [@resolve-js/redux](https://www.npmjs.com/package/@resolve-js/@resolve-js/redux) library that you can use to allow your application's users to post comments and to display these comments on a page.
+
+## Installation
+
+Use the following console input to install the uploader module:
+
+```sh
+yarn add @resolve-js/module-comments
+```
+
+## Register and Configure the Module
+
+Register the installed module in the project's `run.js` file:
+
+```js
+import { merge } from '@resolve-js/scripts'
+import createModuleComments from '@resolve-js/module-comments'
+
+// All of the options specified below are optional.
+const moduleComments = resolveModuleComments({
```

Comment:
It looks like it should be the createModuleComments call instead of resolveModuleComments, see the import expression above

Reply:
Fixed